From 797d2b3792a0d4c4260075c2b05a1aba63715363 Mon Sep 17 00:00:00 2001
From: Ionut Biru <ibiru@archlinux.org>
Date: Wed, 9 May 2012 17:34:10 +0000
Subject: [PATCH] Check for gio to avoid linking issue

/usr/bin/ld: note: 'g_inet_address_new_from_string' is defined in DSO
/usr/lib/libgio-2.0.so.0 so try adding it to the linker command line

Signed-off-by: Ionut Biru <ibiru@archlinux.org>
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 2a6cb2a..461d464 100644
--- a/configure.ac
+++ b/configure.ac
@@ -62,7 +62,7 @@ dnl GTK docs
 GTK_DOC_CHECK
 
 dnl Check for Glib 
-PKG_CHECK_MODULES(GLIB, gobject-2.0 >= 2.30, have_glib=yes, have_glib=no)
+PKG_CHECK_MODULES(GLIB, [gobject-2.0 >= 2.30, gio-2.0 >= 2.30], have_glib=yes, have_glib=no)
 
 if test x$have_glib = xno ; then
     AC_MSG_ERROR([GLib development libraries not found])
-- 
1.7.10.1

